script/Makefile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 VERSIONFILE := pkg/util/defaults/defaults.go
-VERSION ?= 1.6.0-SNAPSHOT
+VERSION := 1.6.0-SNAPSHOT
```

Comment:
Can this not be changed since this is relied upon for using the kustomize installation method I've been working on at https://github.com/apache/camel-k/pull/2284.